CMakeLists.txt

```diff
@@ -13,21 +18,22 @@ set(main_exe_name RigelEngine)
 # Project settings
 ###############################################################################
 
-option(USE_GL_ES "Use OpenGL ES instead of regular OpenGL" OFF)
+option(USE_GL_ES "Use OpenGL ES instead of regular OpenGL" ON)
 option(WARNINGS_AS_ERRORS "Treat compiler warnings as errors" ON)
 
-
 # Dependencies
 ###############################################################################
 
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
+# set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "/home/pratikone/git/emsdk/upstream/emscripten/cmake/Modules/")
```

Comment:
Normally, the path should be set by the toolchain file, so this should not be necessary. There should also not be any harm in adding Rigel's own modules to the path (i.e. `cmake/Modules`), since we won't make use of them when in Emscripten mode. Could you check if it works when you keep the original module path instead of this one?